3-enrich/emr-etl-runner/config/config.yml.sample

```diff
@@ -61,7 +61,7 @@ collectors:
   format: cloudfront # For example: 'clj-tomcat' for the Clojure Collector, 'thrift' for Thrift records, 'tsv/com.amazon.aws.cloudfront/wd_access_log' for Cloudfront access logs or 'ndjson/urbanairship.connect/v1' for UrbanAirship Connect events
 enrich:
   versions:
-    spark_enrich: 1.12.0 # Version of the Spark Enrichment process
+    spark_enrich: 1.13.0 # Version of the Spark Enrichment process
```

Comment:
Just heads-up that we also have `stream_config.yml.sample` now, but on the other hand `spark_enrich` there is useless and subject for removal in future https://github.com/snowplow/snowplow/issues/3711 (up to you to bump it or not)